docs/building.md

```diff
@@ -0,0 +1,15 @@
+# Build from source TL;DR
+
+0. Install Go
+
+1. Get the source 
+
+        $ go get github.com/GoogleCloudPlatform/skaffold
```

Comment:
Since the top level project is not go gettable (the skaffold subcommand is), I'd rather run `go -u -d get github.com/GoogleCloudPlatform/skaffold`

Reply:
Shoudn't this be `go get -u -d github.com/GoogleCloudPlatform/skaffold` ?

Reply:
yes :-) sorry

Reply:
@dgageot @geniass I made changes in place!